site/content/docs/11-deploying.md

```diff
@@ -2,7 +2,21 @@
 title: Deployment
 ---
 
-Sapper apps run anywhere that supports Node 8 or higher.
+Sapper apps run anywhere that supports Node 10 or higher.
+
+### Deploying from sapper build
+
+We will need to have `server.js` dependencies available on the production machine. You can either install the dependencies locally or have the `package.json` and `package-lock.json` available on the production machine and install the dependencies there. Production dependencies can be installed with:
+
+```bash
+npm ci --prod
+```
+
+Make sure to run `npm run build` and have `__sapper__/build` directory generated before proceeding. We will also need to include the `static` directory in the production machine, you can then start your app with:
```

Comment:
> before proceeding.

I'm still not 100% sure about this, is this clear enough to imply that `npm run build` can be installed locally or in the production, just like `npm ci` command, or should it be stated again with something like

```suggestion
Make sure to run `npm run build` and have `__sapper__/build` directory generated, just like with the production dependencies, this can happen locally or on the production machine. We will also need to include the `static` directory in the production machine, you can then start your app with:
```